### ISSUE TITLE
Use renderer component to access size, pointer, renderer... ?

### ISSUE THREAD

**Issue by klevron** [troisjs/trois#40]
TroisJS use the following internal helper : https://github.com/troisjs/trois/blob/master/src/core/useThree.js

It is exposed in components via `three` injection, but it could be better to hide it and add shortcuts to renderer component to be able to access pointer, threejs renderer, size and ?

**Comment by SaFrMo** [comment]
Love this! I was looking for where `onBeforeRender` was moved to and really like this location. Decoupling the render callbacks from `three` itself and moving to the relevant renderer makes a lot of sense.

**Comment by SaFrMo** [comment]
The one edit I can think of would be to change the `provide` name to  `renderer` instead of `rendererComponent`. `rendererComponent` makes me think of the actual canvas DOM element we're rendering to, `renderer` feels like a slightly clearer reference to me. Might just be personal preference though, either way works!

**Comment by klevron** [comment]
Yes a little bit confusing, but also `renderer.three.renderer`, we could use :
- renderer.$renderer : three renderer 
- renderer.$pointer
- renderer.$size

**Comment by klevron** [comment]
Well, It's better to not use `$`, but I am not satisfied :
- renderer.renderer : three renderer
- renderer.three.pointer (can't use renderer.pointer because it's a prop)
- renderer.size
